.github/workflows/release-nightly.yml

```diff
@@ -71,17 +87,15 @@ jobs:
         #
         # NOTE: Using --preid dev.$(git rev-parse --short=7 HEAD) results in `0.24.3-dev.3ddb91d.0+3ddb91d`
         run: |
-          node_modules/.bin/lerna publish --yes --no-verify-access \
-          --canary --dist-tag next --no-git-reset --force-publish \
-          --preid dev --exact
+          node_modules/.bin/lerna publish from-package \
+          --yes \
+          --no-verify-access \
+          --dist-tag next \
+          --no-git-reset \
+          --force-publish \
+          --exact
```

Comment:
no more `--preid dev` ?

Reply:
`preid` flag is not necessary because the version is given, not computed by lerna in the canary logic. The `dev` word in used above when manually crafting the version string

Reply:
Can't comment above for some reason

> Why not take version from https://github.com/ChainSafe/lodestar/blob/master/lerna.json#L7 ?

Should be equivalent indeed, I don't have any preference in grabbing it from there, but the package.json seems more generic long term if we ever switch away from lerna